app/grandchallenge/reader_studies/utils.py

```diff
@@ -17,27 +33,34 @@ def migrate_reader_study_to_display_sets(rs, view_content):  # noqa: C901
             ds = DisplaySet.objects.create(reader_study=rs)
             images = []
             for key in item:
-                image = rs.images.get(name=item[key])
-                images.append(image.pk)
-
+                if "overlay" in key:
+                    continue
                 try:
-                    slug = view_content[key]
+                    slugs = view_content[key]
                 except KeyError:
                     raise ValueError(
                         f"No ComponentInterface provided for {key}."
                     )
-
-                try:
-                    ci = ComponentInterface.objects.get(slug=slug)
-                except ComponentInterfaceValue.DoesNotExist:
+                if len(slugs) == 0:
                     raise ValueError(
-                        f"ComponentInterface {slug} does not exist."
+                        f"No ComponentInterface provided for {key}."
                     )
-
-                civ, _ = ComponentInterfaceValue.objects.get_or_create(
-                    image=image, interface=ci
-                )
-                ds.values.add(civ)
+                else:
+                    if len(slugs) > 2:
+                        logger.warning(
+                            f"More than two interface slugs provided for {key} "
+                            f"in {rs.slug}. Ignoring all values after the second."
+                        )
```

Comment:
Shouldn't this error out instead?

Reply:
I'm not sure. I think it should be ok to provide more values in the view content, but we cannot assign anything to those interfaces, as with the old hanging list there was only the option to provide a main image and an overlay per viewport.

Reply:
These methods should be ultra conservative as in the worst case scenario it's either data loss or wrong data. If you're unsure, error out.